llvm/include/llvm/CodeGen/BasicTTIImpl.h

```diff
@@ -1510,6 +1510,8 @@ class BasicTTIImplBase : public TargetTransformInfoImplCRTPBase<T> {
     Intrinsic::ID IID = ICA.getID();
     if (Function::isTargetIntrinsic(IID))
       return TargetTransformInfo::TCC_Basic;
+    if (IID == Intrinsic::ssa_copy)
+      return TargetTransformInfo::TCC_Free;
```

Comment:
This should be in TargetTransformInfoImpl.h.

Reply:
Ah I see. That is better, thanks.